extras/drift_postgres/lib/src/pg_database.dart

```diff
@@ -54,7 +54,8 @@ class _PgDelegate extends DatabaseDelegate {
   late DbVersionDelegate versionDelegate;
 
   @override
-  Future<bool> get isOpen => Future.value(_openedSession != null);
+  Future<bool> get isOpen =>
+      Future.value(_openedSession != null && !_openedSession!.isOpen);
```

Comment:
Looking at the test failures, should the negation really be here?

Reply:
I'd blame this one on chat-gpt.